webapp/www/greentrip_site/learn_more.html

```diff
@@ -0,0 +1,25 @@
+<html>
+	<head>
+		<div class = "top_bar">
+			<div class = "item" id = "logo"> <a href = "index.html"> <img src = "assets/img/greentrip_logo.png" style = "width:150px; height:100px;"> </a> </div>
+			<div class = "item"> <a href = "incentive.html"> Incentivization </a> </div>
+			<div class = "item"> <a href = "who_we_are.html"> Who We Are </a> </div>
+			<div class = "item"> <a href = "supporters.html"> Supporters </a> </div>
+			<div class = "item"> <a href = "related_work.html"> Related Works </a> </div>
+			<div class = "button"> JOIN STUDY </div>
+		</div>
+	<link rel="stylesheet" type="text/css" href="assets/css/index.css">
+	<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=EB+Garamond">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+	</head>
+	<div class = "big_box" id = "second_page"> 
+		<div class = "large_text" id = "large_text_1"> Learn More </div>
+		<div class = "smaller_text"> <i class="fas fa-leaf"></i> Greentrip is a study examining how sustainable suggestions affect travel behavior.  </div>
```

Comment:
I wonder if we should say this up front. Because if you are part of the control group, none of this is going to apply to you, and then you will know that you are in the control group. How do you avoid that?